screensaver.video/addon.xml

```diff
@@ -1,13 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="screensaver.video" name="VideoScreensaver" version="1.3.0" provider-name="robwebset">
+<addon id="screensaver.video" name="VideoScreensaver" version="1.4.0" provider-name="robwebset">
 	<requires>
 		<import addon="xbmc.python" version="2.14.0"/>
 		<import addon="script.module.simplejson" version="3.3.0"/>
 	</requires>
 	<extension point="xbmc.ui.screensaver" library="default.py"/>
-  	<extension point="xbmc.python.script" library="screensaver.py" />
-	<extension point="xbmc.service" library="service.py" start="login">
-		<provides>executable</provides>
+  	<extension point="xbmc.python.script" library="screensaver.py"/>
+	<extension point="xbmc.service" library="service.py" start="login"/>
+	<extension point="xbmc.python.pluginsource" library="plugin.py">
+		<provides>video</provides>
```

Comment:
I have to say, I like that all the related features are together in one addon - I've done this in several addons.  All the features are related, and normally depend on each other.  I think if some-one chooses to use the addon then it's OK to ensure they have all of the addon.